<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNew</class>
 <widget class="QDialog" name="AddNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>611</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="verticalSpacing">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>14</number>
      </property>
      <item row="0" column="1">
       <widget class="QLineEdit" name="name_lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="place_lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Место</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="generate_Button">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 85, 0);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>сгенерировать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="psw_lineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QPushButton" name="save_Button">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 32, 66);</string>
        </property>
        <property name="text">
         <string>сохранить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exit_Button">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 85, 0);</string>
        </property>
        <property name="text">
         <string>отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="back_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>449</width>
     <height>449</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../build/img/img.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>260</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: transfert</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="where_label">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>320</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: transfert</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="psw_label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>400</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:transfert</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="genPsw_groupBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>400</width>
     <height>370</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>370</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>370</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(19, 0, 36, 251);
font: 600 13pt &quot;Segoe UI Variable Display Semib&quot;;
</string>
   </property>
   <property name="title">
    <string>             сгенерировать пароль</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>271</width>
      <height>293</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="capitalLetters_checkBox">
       <property name="styleSheet">
        <string notr="true">font: 600 9pt &quot;Segoe UI Variable Display Semib&quot;;</string>
       </property>
       <property name="text">
        <string>Заглавные буквы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="caseLow_checkBox_2">
       <property name="styleSheet">
        <string notr="true">font: 600 9pt &quot;Segoe UI Variable Display Semib&quot;;</string>
       </property>
       <property name="text">
        <string>Строчные буквы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_num">
       <property name="styleSheet">
        <string notr="true">font: 600 9pt &quot;Segoe UI Variable Display Semib&quot;;</string>
       </property>
       <property name="text">
        <string>Цифры</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_sym">
       <property name="styleSheet">
        <string notr="true">font: 600 9pt &quot;Segoe UI Variable Display Semib&quot;;</string>
       </property>
       <property name="text">
        <string>Символы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>5</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(21, 18, 53);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="numLig_label">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 600 9pt &quot;Segoe UI Variable Display Semib&quot;;</string>
         </property>
         <property name="text">
          <string>длина</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="startGen_pushButton">
         <property name="styleSheet">
          <string notr="true">font: 600 9pt &quot;Segoe UI Variable Display Semib&quot;;
background-color: rgb(85, 0, 127);</string>
         </property>
         <property name="text">
          <string>сгенерировать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 85, 0);
color: rgb(255, 255, 255);
font: 600 9pt &quot;Segoe UI Variable Display Semib&quot;;
</string>
         </property>
         <property name="text">
          <string>Закрыть</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(21, 18, 53);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
  </widget>
  <zorder>where_label</zorder>
  <zorder>name_label</zorder>
  <zorder>psw_label</zorder>
  <zorder>back_label</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>genPsw_groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
